dev_tools/qualtran_dev_tools/reference_docs.py

```diff
@@ -120,11 +120,11 @@ def __init__(self, page_info):
         )
 
 
-class MyFunctionPageBuilder(mixin_custom_template('function'), FunctionPageBuilder):
+class MyFunctionPageBuilder(mixin_custom_template('function'), FunctionPageBuilder):  # type: ignore[misc]
     """Use a custom template for function pages."""
 
 
-class MyTypeAliasPageBuilder(mixin_custom_template('type_alias'), TypeAliasPageBuilder):
+class MyTypeAliasPageBuilder(mixin_custom_template('type_alias'), TypeAliasPageBuilder):  # type: ignore[misc]
```

Comment:
for my edification: does `[misc]` refer to a specific mypy error? class of error?

Reply:
misc is for a bunch of random stuff that are relatively rare errors.  In this case, it is because mypy doesn't allow dynamically created mix-ins as base classes.  My understanding is that this is fine but is a limitation of mypy, but this is starting to get into territory I am not super familiar with.

Reply:
Makes sense. This is in the dev_tools, so less essential to keep everything perfect and static